exe-common/src/sync.rs

```diff
@@ -167,16 +164,16 @@ fn net_sync_to<A: Api>(
     let mut chain_state = chain_state::restore_chain_state(
         storage,
         genesis_data,
-        if our_tip.1 {
-            &our_tip.0.parent
+        if our_tip_is_genesis {
+            &our_tip.parent
         } else {
-            &our_tip.0.hash
+            &our_tip.hash
         },
     )?;
```

Comment:
Just to be extra sure - can someone please clarify why the **parent** is used here if the tip is a genesis? 🤔Is this "genesis" refers to EBB?

Reply:
Ah yes, seems obvious to be EBB, since `BlockDate::Boundary` is used. My bad.